app/assets/stylesheets/application.css.scss

```diff
@@ -64,6 +78,13 @@ td, th {
   }
 }
 
+.dashboard-cursive{
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.